ci_framework/roles/rhol_crc/tasks/main.yml

```diff
@@ -0,0 +1,47 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Check RHOL/CRC binary exists
+  ansible.builtin.stat:
+    path: "{{ cifmw_rhol_crc_binary }}"
+  register: cifmw_rhol_crc_binary_stat
+
+- name: Get RHOL/CRC binary if does not exist
+  ansible.builtin.include_tasks: tasks/binary.yml
```

Comment:
no need to prefix with `tasks/` - directly include the taskfile, it's at the same level.

Reply:
My mistake, I was doing tests from CLI and I forgot to delete it.